santa-tracker-base.conf

```
@@ -27,16 +27,12 @@ base {
     ]
 }
 
-enableInstantExecution = [
-    "-Dorg.gradle.unsafe.instant-execution=true",
-    "-Dorg.gradle.unsafe.instant-execution.fail-on-problems=false"
+enableConfigurationCache = [
+    "--configuration-cache=warn"
 ]
 
-enablePartialInvalidation = ["-Dorg.gradle.unsafe.vfs.partial-invalidation=true"]
-
-enableVfsRetention = [
-    "-Dorg.gradle.unsafe.vfs.retention=true",
-    "-Dorg.gradle.unsafe.vfs.changes="${file-to-change}
+enableFileWatching = [
```

```suggestion
enableFileSystemWatching = [
```

Same here, let's not use `file watching`.